cgeo tests do not compile anymore

### Describe your problem!

```
11:29:18  > Task :main:compileBasicDebugAndroidTestJavaWithJavac
11:29:18  /home/jenkins/slave/workspace/cgeo continuous integration/tests/src-android/cgeo/geocaching/filters/GeocacheFilterTestUtils.java:75: error: no suitable method found for getBatchOfStoredCaches(<null>,CacheType,int,GeocacheFilter,<null>,boolean,int)
11:29:18              sr = DataStore.getBatchOfStoredCaches(null, CacheType.ALL, -1, gcFilter, null, false, 1);
11:29:18                            ^
11:29:18      method DataStore.getBatchOfStoredCaches(Geopoint,int) is not applicable
11:29:18        (actual and formal argument lists differ in length)
11:29:18      method DataStore.getBatchOfStoredCaches(Geopoint,int,GeocacheFilter,CacheComparator,boolean,int) is not applicable
11:29:18        (actual and formal argument lists differ in length)
```

### How to reproduce?

This happened as we have merged two incompatible PRs, #11535 and #11322, at the same time.

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Yes

### c:geo Version

master

### System information

_No response_

### Additional Information

_No response_

--- a/main/src/cgeo/geocaching/maps/CGeoMap.java
+++ b/main/src/cgeo/geocaching/maps/CGeoMap.java
@@ -11,7 +11,6 @@ import cgeo.geocaching.activity.Progress;
 import cgeo.geocaching.connector.ConnectorFactory;
 import cgeo.geocaching.connector.gc.Tile;
 import cgeo.geocaching.downloader.DownloaderUtils;
-import cgeo.geocaching.enumerations.CacheType;
 import cgeo.geocaching.enumerations.CoordinatesType;
 import cgeo.geocaching.enumerations.LoadFlags;
 import cgeo.geocaching.enumerations.LoadFlags.RemoveFlag;

--- a/tests/src-android/cgeo/geocaching/filters/GeocacheFilterTestUtils.java
+++ b/tests/src-android/cgeo/geocaching/filters/GeocacheFilterTestUtils.java
@@ -1,7 +1,6 @@
 package cgeo.geocaching.filters;
 
 import cgeo.geocaching.SearchResult;
-import cgeo.geocaching.enumerations.CacheType;
 import cgeo.geocaching.enumerations.LoadFlags;
 import cgeo.geocaching.filters.core.AndGeocacheFilter;
 import cgeo.geocaching.filters.core.BaseGeocacheFilter;
@@ -72,7 +71,7 @@ public class GeocacheFilterTestUtils {
         final SearchResult sr;
         try {
             DataStore.storeIntoDatabase(cache);
-            sr = DataStore.getBatchOfStoredCaches(null, CacheType.ALL, -1, gcFilter, null, false, 1);
+            sr = DataStore.getBatchOfStoredCaches(null, -1, gcFilter, null, false, 1);
 
         } finally {
             DataStore.removeCache(cache.getGeocode(), EnumSet.of(LoadFlags.RemoveFlag.CACHE, LoadFlags.RemoveFlag.DB));

